helm-chart/renku-gateway/templates/_helpers.tpl

```diff
@@ -51,3 +51,16 @@ Hack for calling templates in a fake scope (until this is solved https://github.
 {{- $template := index . 2 }}
 {{- include $template (dict "Chart" (dict "Name" $subchart) "Values" (index $dot.Values $subchart) "Release" $dot.Release "Capabilities" $dot.Capabilities) }}
 {{- end }}
+
+{{/*
```

Comment:
ah nice, I like the idea to put this into a separate helper.

Reply:
Yeah seeing that `call-nested` template in a few places scares me a bit.